client/src/AppRouter.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Route, HashRouter, Switch } from 'react-router-dom';
+import App from './App';
+import { Home, Login, Register } from './components';
+
+const AppRouter = () => {
```

Comment:
why delete Router.jsx

Reply:
Right I should have just asked about that, but it's exporting the component AppRouter so why shouldn't it be named that?

Reply:
ok

Reply:
but when you merge, make sure AppRouter keeps the latest routes